go/pkg/client/bytestream.go

```diff
@@ -73,6 +73,61 @@ func (c *Client) writeChunked(ctx context.Context, name string, ch *chunker.Chun
 	return totalBytes, err
 }
 
+// WriteBytesWithOffset uploads a byte slice.
+func (c *Client) WriteBytesWithOffset(ctx context.Context, name string, data []byte, opts *ByteStreamWriteOpts) (int64, error) {
+	ue := uploadinfo.EntryFromBlob(data)
+	ch, err := chunker.New(ue, false, int(c.ChunkMaxSize))
+	if err != nil {
+		return 0, err
+	}
+	return c.writeChunkedWithOffset(ctx, name, ch, opts)
+}
+
+// writeChunkedWithOffset uploads chunked data with a given resource name to the CAS at an arbitrary offset.
```

Comment:
I think this comment is worth having for the exported method to expose it to IDEs and users.

Reply:
Updated the exported method's comment.